x/deployment/types/msgs.go

```diff
@@ -7,9 +7,9 @@ import (
 
 // msg types
 const (
-	TypeMsgCreateDeployment = "create_deployment"
-	TypeMsgUpdateDeployment = "update_deployment"
-	TypeMsgCloseDeployment  = "close_deployment"
+	typeMsgCreateDeployment = "create_deployment"
+	typeMsgUpdateDeployment = "update_deployment"
+	typeMsgCloseDeployment  = "close_deployment"
```

Comment:
Either

```go
// msgs.go
typeMsgCreateDeployment = ModuleName + "/msg-create-deployment"

// codec.go
cdc.RegisterConcrete(MsgCreateDeployment{}, typeMsgCreateDeployment, nil)
```

or
```go
// msgs.go
typeMsgCreateDeployment = msg-create-deployment"

// codec.go
cdc.RegisterConcrete(MsgCreateDeployment{}, ModuleName + "/" + typeMsgCreateDeployment, nil)
```
Same goes for the rest of these.

Reply:
if you check the godoc for the `RegisterConcrete` the name must match the type name

Reply:
great then use

```go
// msgs.go
typeMsgCreateDeployment = `MsgCreateDeployment`

// codec.go
cdc.RegisterConcrete(MsgCreateDeployment{}, ModuleName + "/" + typeMsgCreateDeployment, nil)
```

I feel like the request was pretty obvious, I don't know why we're still talking about it.

Reply:
Events have different formats (they use snake case) than amino registration, hence the difference. 

> I feel like the request was pretty obvious, I don't know why we're still talking about it.

I totally agree, idk why are you still nitpicking on these small things...

Reply:
> I totally agree, idk why are you still nitpicking on these small things...

because you resolved the conversation without saying why you didn't address it.

Reply:
Can you explain why the snake case is necessary?  The [documentation](https://github.com/cosmos/cosmos-sdk/blob/6a9c8bcb781c10ebe27beea75dfa49e19c94add9/types/tx_msg.go#L17-L18) for this method says:

> Returns a human-readable string for the message, intended for utilization within tags

It doesn't say anything about snake case.